### PR TITLE
Release 4.18

### PR DESCRIPTION
**Pull request by new4762**
#### 1. Objective

Release information for version 4.18

#### 🚀 Enhancements
- Add support for BBL Mobile Banking and BAY Mobile Banking (PR [#252](https://github.com/omise/omise-woocommerce/pull/252))

#### 3. Quality assurance

All test are done in each PR, refer to each for more info.

**✏️ Details:**

Test can be done locally with wordpress and manual installation of omise-woocommerce

#### 4. Impact of the change

No impact

#### 5. Priority of change

Normal

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # CHANGELOG
+### [v4.18 _(Mar 8, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.18)
```
Comment (hpariwat) [inline CHANGELOG.md:2]: Please also update the date too.

Comment (new4762) [reply CHANGELOG.md]: NIce catch fixed -> https://github.com/omise/omise-woocommerce/pull/254/commits/7851ff5c67e9a52eca10ba088ff00cf00b43ce69 🙏

**Review by tookaa2 — APPROVED** [review]

File: readme.txt [hunk 2]
```diff
@@ -33,10 +33,10 @@ From there:
 
 == Changelog ==
 
-= 4.17.1 =
+= 4.18 =
 
-#### 👾 Bug Fixes
-- Fix issue Mobile Banking and Rabbit LINE Pay not showing.
```
Comment (kan98) [inline readme.txt:39]: maybe we shouldn't remove the changelog of an older release, rather we should add to it

Comment (new4762) [reply readme.txt]: didn't see this got replaced as well
 just create branch base on v4.17 so after merge 4.17.1 is got replace by 4.18
nice catch 👍  
Fixed -> https://github.com/omise/omise-woocommerce/pull/254/commits/778ea0b1b6a14c945f3d6b21011ee009060c713e

**Review by kan98 — APPROVED** [review]

**Review by hpariwat — APPROVED** [review]

**Review by FhanOmise — APPROVED** [review]

**Pull request merged**